buf.yaml

```diff
@@ -0,0 +1,87 @@
+      - vault/hcp_link/proto/meta/meta.proto
+      - vault/hcp_link/proto/node_status/status.proto
+    IMPORT_USED:
+      - vault/hcp_link/proto/node_status/status.proto
+    PACKAGE_DIRECTORY_MATCH:
+      - helper/forwarding/types.proto
+      - helper/identity/mfa/types.proto
+      - helper/identity/types.proto
+      - helper/storagepacker/types.proto
+      - physical/raft/types.proto
+      - sdk/database/dbplugin/database.proto
+      - sdk/database/dbplugin/v5/proto/database.proto
+      - sdk/helper/pluginutil/multiplexing.proto
+      - sdk/logical/event.proto
+      - sdk/logical/identity.proto
+      - sdk/logical/plugin.proto
+      - sdk/logical/version.proto
+      - sdk/plugin/pb/backend.proto
+      - vault/activity/activity_log.proto
+      - vault/activity/generation/generate_data.proto
+      - vault/hcp_link/proto/link_control/link_control.proto
+      - vault/hcp_link/proto/meta/meta.proto
+      - vault/hcp_link/proto/node_status/status.proto
+      - vault/tokens/token.proto
+    PACKAGE_VERSION_SUFFIX:
+      - helper/forwarding/types.proto
+      - helper/identity/mfa/types.proto
+      - helper/identity/types.proto
+      - helper/storagepacker/types.proto
+      - physical/raft/types.proto
+      - sdk/database/dbplugin/database.proto
+      - sdk/helper/pluginutil/multiplexing.proto
+      - sdk/logical/event.proto
+      - sdk/logical/identity.proto
+      - sdk/logical/plugin.proto
+      - sdk/logical/version.proto
+      - sdk/plugin/pb/backend.proto
+      - vault/activity/activity_log.proto
+      - vault/activity/generation/generate_data.proto
+      - vault/hcp_link/proto/link_control/link_control.proto
+      - vault/hcp_link/proto/meta/meta.proto
+      - vault/hcp_link/proto/node_status/status.proto
+      - vault/request_forwarding_service.proto
+      - vault/tokens/token.proto
+    RPC_REQUEST_RESPONSE_UNIQUE:
+      - sdk/database/dbplugin/database.proto
+      - sdk/database/dbplugin/v5/proto/database.proto
+      - sdk/plugin/pb/backend.proto
+    RPC_REQUEST_STANDARD_NAME:
+      - sdk/database/dbplugin/database.proto
+      - sdk/database/dbplugin/v5/proto/database.proto
+      - sdk/logical/version.proto
+      - sdk/plugin/pb/backend.proto
+      - vault/request_forwarding_service.proto
+    RPC_RESPONSE_STANDARD_NAME:
+      - sdk/database/dbplugin/database.proto
+      - sdk/database/dbplugin/v5/proto/database.proto
+      - sdk/logical/version.proto
+      - sdk/plugin/pb/backend.proto
+      - vault/hcp_link/proto/meta/meta.proto
+      - vault/request_forwarding_service.proto
+    SERVICE_SUFFIX:
+      - sdk/database/dbplugin/database.proto
+      - sdk/database/dbplugin/v5/proto/database.proto
+      - sdk/helper/pluginutil/multiplexing.proto
+      - sdk/logical/version.proto
+      - sdk/plugin/pb/backend.proto
+      - vault/hcp_link/proto/link_control/link_control.proto
+      - vault/hcp_link/proto/meta/meta.proto
+      - vault/request_forwarding_service.proto
```

Comment:
This is used to ignore all existing lint warnings, which means `buf lint` returns no warnings on the existing files.